ftrack_s3_accessor/s3.py

```diff
@@ -287,3 +287,13 @@ def make_container(self, resource_identifier, recursive=True):
 
         s3_object = self.bucket.Object(resource_identifier)
         s3_object.put(Body='')
+
+    def get_url(self, resource_identifier=None):
+        '''Return url for *resource_identifier*.'''
+        s3_object = self.bucket.Object(resource_identifier)
+        try:
+            location = boto3.client('s3').get_bucket_location(Bucket=self.bucket_name)['LocationConstraint']
+            url = "https://s3-{}.amazonaws.com/{}/{}".format(location, self.bucket_name, s3_object.key)
```

Comment:
```suggestion
            url = f"https://s3-{location}.amazonaws.com/{self.bucket_name}/{s3_object.key}"
```
Is there any reason not to use 3.6+ fstrings here?

Reply:
we usually prefer to stick to older syntax for now, but being this your repo I'm happy to use newer syntaxes.